tests/test_fatigue.py

```diff
@@ -0,0 +1,240 @@
+import pytest
+import numpy as np
+from bioptim import OdeSolver
+
+from .utils import TestUtils
+
+import biorbd_casadi as biorbd
+
+from bioptim.misc.enums import Fatigue
+from bioptim import (
+    OptimalControlProgram,
+    ObjectiveList,
+    ObjectiveFcn,
+    Dynamics,
+    DynamicsFcn,
+    QAndQDotBounds,
+    InitialGuess,
+    Objective,
+    OdeSolver,
+    Bounds,
+    Constraint,
+    ConstraintFcn,
+    XiaTorqueFatigue,
+    XiaFatigueDynamicsList,
+    XiaFatigueStateInitialGuess,
+    XiaFatigueStateBounds,
+    XiaFatigueControlsBounds,
+    XiaFatigueControlsInitialGuess,
+    Node,
+    Axis,
+)
+
+
+def prepare_ocp_static_arm(
+    biorbd_model_path: str,
+    final_time: float,
+    n_shooting: int,
+    ode_solver: OdeSolver = OdeSolver.RK4(),
+    with_residual_torque: bool = False,
+    fatigue: list = None,
+) -> OptimalControlProgram:
+    """
+    Prepare the ocp
+    Parameters
+    ----------
+    biorbd_model_path: str
+        The path to the bioMod
+    final_time: float
+        The time at the final node
+    n_shooting: int
+        The number of shooting points
+    ode_solver: OdeSolver
+        The ode solver to use
```

Comment:
Is this a copy of the example file? If so, just refer to the file so like that we know the examples work